addons/web/static/src/views/fields/state_selection/state_selection_field.js

```diff
@@ -36,7 +36,9 @@ export class StateSelectionField extends Component {
             for (const [index, [value, label]] of this.options.entries()) {
                 useCommand(
                     sprintf(this.env._t("Set kanban state as %s"), label),
-                    () => this.updateRecord(value),
+                    () => {
+                        this.updateRecord(value);
+                    },
```

Comment:
Should not return because it doesn't this useCommand doesn't expect next set of commands.